platform/ios/jazzy.yml

```diff
@@ -51,6 +51,21 @@ custom_categories:
       - MGLPolygonFeature
       - MGLPolylineFeature
       - MGLShapeCollectionFeature
+  - name: Runtime Styling
```

Comment:
Runtime styling is an (intentionally) awkward public name for a feature. "Style Layers" and "Data Sources", as two separate categories, would better differentiate these classes from the other categories.